charts/zipkin/values.yaml

```diff
@@ -143,3 +143,12 @@ extraServicePorts: []
 
 # global variables to use in values. typically used in tests.
 global: {}
+
+# extra manifests to deploy with the helm chart
+extraManifests: []
```

Comment:
mind adding to values schema and the top-level README?

Reply:
Done, added to both readme and schema file